src/services/subject.js

```
@@ -0,0 +1,33 @@
+const Subject = require('~/models/subject');
+
+const subjectService = {
+  getSubjects: async ({ search = '', page = 1, limit = 20 } = {}) => {
+    const query = {};
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)